combat/combatSystem.cpp

```diff
@@ -373,6 +378,144 @@ int Monster::getWeaponSkill(const std::shared_ptr<Object>  weapon) const {
     return(weaponSkill);
 }
 
+//**********************************************************************
+//                      getClassWeaponskillBonus
+//**********************************************************************
+
+int Player::getClassWeaponskillBonus(const std::shared_ptr<Object>  weapon) const {
```

Comment:
```suggestion
int Player::getClassWeaponSkillBonus(const std::shared_ptr<Object>  weapon) const {
```
References would have to be updated as well

Reply:
new function so never been called til now